Bp 31 implement contact us

jambao24 opened the pull request:
Implemented basic design elements specified in ticket 31 and removed "ContactInfo" from bottom of each page.

blob:https://minmer.atlassian.net/0736bd44-3134-46cd-b9a4-e26d11ffbd30#media-blob-url=true&id=94ceb90a-8279-4b53-935b-1a84bc309908&collection=&contextId=10030&mimeType=image%2Fpng&name=image-20200730-034324.png&size=439246&width=947&height=638

![BP-31_attempt](https://user-images.githubusercontent.com/34732378/91247657-9f341780-e707-11ea-8457-c47d67da540e.PNG)

Commit:
Added design for the homepage.

Commit:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

Commit:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

Commit:
Added new design for the website.

jambao24 committed:
Update Navigation.module.scss

jambao24 committed:
Revert "Update Navigation.module.scss"

This reverts commit 1199540ef63df8b07f46150151dbd58e1a2b1fea.

jambao24 committed:
Revert "updated top bar colors"

This reverts commit 67ac0c8f8a6a9b0c96049d1f90fb12b656821ad2.

jambao24 committed:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

jambao24 committed:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

jambao24 committed:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

Commit:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

# Conflicts:
#	src/components/FeaturedServices/FeaturedServices.module.scss

MinhLu12 committed:
Remake Summer's MeetDrBergin component

Commit:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

Commit:
Merge branch 'BP-30_ImplementMeetDrBerginPage' of https://github.com/MinhLu12/bergin-prosthodontics into BP_30-ImplementMeetDrBerginPage

Commit:
Added logos to the MeetDrBergin page.

Commit:
Added some routing

MinhLu12 committed:
Fix all routing, not aesthetic

MinhLu12 committed:
Make routes look a bit prettier

Still WIP, hyperlinks are still blue, and the "Home" text is weirdly lighter colored even though it's black like the other ones.

jambao24 committed:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

jambao24 committed:
added ContactUs page

added separate page for ContactUs, clickable link in "CONTACT US" button. Still need to fix column formatting, left column text size + spacing, and add a text box inside the Google Maps window

jambao24 committed:
Google maps box contains address text box now

still trying to figure out how to get the columns working properly

jambao24 committed:
major restructuring is done

still tweaking minor details like sizes and position

jambao24 committed:
updated position of items per JIRA design

ContactInfo content still present at bottom of each page

MinhLu12 committed:
Remove tabs that are not needed

jambao24 committed:
duplicate "ContactInfo" at bottom removed

MinhLu12 commented:
Good start! A few things here:
1. This doesn't look like the design. The spacing, background color, and texts, fonts aren't correct. Please refer to the design posted in the JIRA ticket
![image](https://user-images.githubusercontent.com/13816671/91250147-2c2d9f80-e70d-11ea-9249-c28bc2f284d4.png)

2. Please pull from master, so that we can get a pull request of only your changes. As it is, there's 20 files.
3. There are merge conflicts as well.

jambao24 committed:
Merge branch 'master' of https://github.com/MinhLu12/bergin-prosthodontics

jambao24 committed:
changed background color for ContactUs

jambao24 committed:
Can't figure out line spacing yet

jambao24 committed:
all changes done aside from "Contact Us" font

can't find the font corresponding to the one shown in the design image in "global.scss"

jambao24 committed:
Merge branch 'RemoveTabs'

jambao24 committed:
Merge branch 'master' into BP-31-Implement_ContactUs

jambao24 committed:
realized everything in the design for Column_Left has the same font

MinhLu12 commented:
Looks a lot better!

However, the color of the text in the Contact Us button is off
![image](https://user-images.githubusercontent.com/13816671/91759300-4b518480-eb86-11ea-9855-5a00ea414d14.png)

And also don't remove the Contactinfo.js please, that should be on every page except the ContactUs.js page. We can call about this if you want!

jambao24 commented:
> Looks a lot better!
> 
> However, the color of the text in the Contact Us button is off
> ![image](https://user-images.githubusercontent.com/13816671/91759300-4b518480-eb86-11ea-9855-5a00ea414d14.png)
> 
> And also don't remove the Contactinfo.js please, that should be on every page except the ContactUs.js page. We can call about this if you want!

Ok, got it. I just merged all the changes from the master version after merging "Remove tabs" into master.

MinhLu12 commented:
`<Route
                    render={({ location }) =>
                        location.pathname !== "/contact" ? (
                            <ContactInfo />
                        ) : null
                    }
                />`

Put this as an additional route outside of the `Switch` in App.js. This will check the pathname and conditionally render the ContactInfo

jambao24 committed:
fixed remaining issues

1) modified "Contact Us" button format to match "Meet Dr. Bergin" button
2) added a new folder "CopyrightInfo" and split out the Copyright line into a separate file

MinhLu12 commented on src/components/CopyrightInfo/CopyrightInfo.module.scss:
```diff
@@ -0,0 +1,47 @@
+@import 'global.scss';
```
@jambao24 Just as a note, we shouldn't blindly copy and paste SCSS. For Copyrightinfo, we only need the `Copyright` css class, and nothing else.

MinhLu12 commented on src/App.js:
```diff
@@ -13,15 +15,15 @@ export default function App() {
 
-                <ContactInfo></ContactInfo>
+                <Route render={({ location }) => location.pathname !== "/contact" ? (<ContactInfo />) : null} />
+                <CopyrightInfo></CopyrightInfo>
```
IMO the copy right info is too small to break apart into its own component, but this is totally fine!

MinhLu12 commented on src/pages/contact/ContactUs.module.scss:
```diff
@@ -0,0 +1,96 @@
+@import 'global.scss';
+
+.Contact_Us {
```
.FeaturedServices, .Text, .Break, .Images, .Caption and among others are not used here. Copying and pasting has its place, but if you do it, please remove code that's not used!

MinhLu12 committed:
Remove unneeded SCSS, fix text sizes

MinhLu12 committed:
Revert CopyRightInfo component

MinhLu12 committed:
Fix remaining spacing issues, make map smaller to align

MinhLu12 commented on src/components/ContactInfo/ContactInfo.js:
```diff
@@ -35,10 +35,11 @@ const ContactInfo = () => {
 
-            <div className={cn.Copyright}>
-                © 2020 Junping Bergin PLLC, Incorporated. All Rights Reserved.
+                <div className={cn.Copyright}>
```
I put Copyright back to where it was since it's too small to break it up - it'd be too fragmented otherwise! And the likelihood that we need code changes in this single <div> is very small.

jambao24 replied:
I put Copyright in a separate class because you said you wanted it to be visible in the Contact Us tab. Now it's only visible on the other pages and it isn't on the bottom of the page anymore. 
![ContactBar_0905](https://user-images.githubusercontent.com/34732378/92313467-d4026300-ef80-11ea-9b5f-f7b6d81c9b2f.PNG)
![ContactUs_page_0905](https://user-images.githubusercontent.com/34732378/92313468-d49af980-ef80-11ea-9c8c-e4c64a0226fc.PNG)

MinhLu12 commented on src/pages/contact/ContactUs.js:
```diff
@@ -0,0 +1,57 @@
+const ContactUs = () => {
+    return (
+        <>
+            <div className={cn.Contact_Us}>
```
I made a bunch of smaller HTML and CSS changes to align everything
